### PR TITLE
Landing page changes

### PR DESCRIPTION
**Pull request by CMorenoStokoe**
Index and user-guide changes:
- **overhauled layout to fix multiple issues of overlapping elements on mobile**
- **added favicons**
- **added data source information and links**
- **fixed welsh title**
- **fixed image positions**
- 2 add link for people to report missing links
- 4 added data sources acknowledgements
- 5 add note about reloading the page
- 10 added note on openstreetmap and improve link
- adjusted text to be more legible
- removed container causing layout issues
- fixed break points on mobile
- added drop shadow to make title text more legible particularly on mobile
- 'go to dashboard' changed to 'go to map' on user guide
- Al suggested to remove the logo shadow because on the landing page it doesn't look as good
- added 'read about' text and button to title heading section on index because it was not clear you could scroll down before
- removed arrow images because they would not re-alight to new organisations and point to empty space or the wrong elements
- reintroduced color consistency which a previous merge knocked out
- reorganised block of text on index for better design on desktop
- added title font to buttons for legibility and consistency of design
- edited image PNGs to be centered
- changed image layout to be more centered and spaced
- changed wording in the user-guide with suggestion from Nina
- reworked about button on index page

### CODE REVIEW

File: user-guide.html [hunk 1]
```diff
@@ -254,28 +261,31 @@ <h1 class="font-head">Our data</h1><br>
...
           <div class="content-container">
             <h1 class="font-head">Data layers</h1><br>
             <p>There are two types of data layers:</p>
               <ul>
-                <li><p><strong>Risk factors:</strong> Factors which indicate that a community may be at elevated risk during the COVID-19 pandemic</p></li>
-                <li><p><strong>Community resources:</strong> Factors which indicate that a community may have better resources for coping during COVID-19 pandemic</p></li>
+                <li><p><strong>Local need:</strong> Factors which indicate that a community may be at elevated risk during the COVID-19 pandemic</p></li>
+                <li><p><strong>Local support:</strong> Factors which indicate that a community may have better resources for coping during COVID-19 pandemic</p></li>
               </ul>
+            <p>We are using covid mutual aid database to identify community support groups, if you don't your local group on the map
```
Comment (ninadicara) [inline user-guide.html:275]: Could I suggest a change to the wording here? 
"We are using the Police Rewired community support dataset (https://www.policecoders.org/home/covid-19) to map community support groups. If you don't see your local group then you can add it by registering on the Covid Mutual Aid website. (https://covidmutualaid.org/)"

File: user-guide.html [hunk 2]
```diff
@@ -118,39 +121,39 @@ <h5 class="modal-title font-head" id="exampleModalLongTitle">Data catalogue</h5>
...
       <!-- Left content -->
-      <div class="col-sm mainContent">
+      <div class="col-xl-5 mainContent">
         <div class="content-container">
           <h1 class="font-head-light">User guide</h1>
           <br>
           <h4 id="subheading">This is a short guide to the dashboard and how to use it 
             to identify vulnerable communities in Wales
             <br><br>
             A full guide can be found in the <a href="https://github.com/DynamicGenetics/Covid-Communities-Map/wiki">Wiki</a>
```
Comment (ninadicara) [inline user-guide.html:134]: Suggested wording change: 
"Full documentation on our methodology and datasources can be found..."

**Comment by CMorenoStokoe** [comment]
> Hi Chris, have checkout out locally and had a browse :) I had a couple of requests/suggestions if that's okay. I also have some thoughts about the information we should provide on the User Guide but I'll leave that for a later time when we have the final design etc.
> 
> I've left two wording suggestion changes as comments and then have two queries below ->
> 
> The first part of the landing page had a lot of buttons, but some of them are the same (e.g. Get Started is also the User Guide, and there are two 'About' buttons, as well as the click down arrow which also takes you to the 'About' section). Could the navigation bar the top right hand corner be displayed in the center of that tile of text instead to avoid repeating buttons? Not sure what you think!
> 
> Screenshot 2020-05-23 at 08 45 01
> 
> Second thing (sorry not a comment in the code because I'm not sure where it is) but some of the images in the User Guide seem to be aligned along the top instead of centered ->
> 
> Screenshot 2020-05-23 at 09 54 05
> 
> Thanks,
> Nina

Redundancy is an important part of design because you account for different user modalities (everyone has a different way of browsing the web: some will use the navlinks, while others will look for the big shiny get started button). I don't think the extra buttons significantly affect either the design or the user interaction so I don't think a change is necessary here. What do you think?

**Comment by CMorenoStokoe** [comment]
> Second thing (sorry not a comment in the code because I'm not sure where it is) but some of the images in the User Guide seem to be aligned along the top instead of centered ->

This is a good point and the reason you couldn't locate the issue is actually because this is spacing in the images themselves. I will action on this: I'll now go through and center each one manually.

**Comment by ninadicara** [comment]
> > The first part of the landing page had a lot of buttons, but some of them are the same (e.g. Get Started is also the User Guide, and there are two 'About' buttons, as well as the click down arrow which also takes you to the 'About' section). Could the navigation bar the top right hand corner be displayed in the center of that tile of text instead to avoid repeating buttons? Not sure what you think!
> 
> Hi Nina, thanks. Redundancy is an important part of design because you account for different user modalities (everyone has a different way of browsing the web: some will use the navlinks, while others will look for the big shiny get started button). I don't think the extra buttons significantly affect either the design or the user interaction so I don't think a change is necessary here. What do you think?

Yup that makes sense! Instead would it be possible for the extra 'about' button to look like the 'Get Started' button and sit just the right of it, instead of with extra text under the image?

File: user-guide.html (hunk 1 above)
Comment (CMorenoStokoe) [reply user-guide.html:276]: Perfect thank you that's much nicer wording - done.

File: user-guide.html (hunk 2 above)
Comment (CMorenoStokoe) [reply user-guide.html:134]: Thank you again, nicer wording- done.

**Comment by CMorenoStokoe** [comment]
> > > The first part of the landing page had a lot of buttons, but some of them are the same (e.g. Get Started is also the User Guide, and there are two 'About' buttons, as well as the click down arrow which also takes you to the 'About' section). Could the navigation bar the top right hand corner be displayed in the center of that tile of text instead to avoid repeating buttons? Not sure what you think!
> > 
> > 
> > Hi Nina, thanks. Redundancy is an important part of design because you account for different user modalities (everyone has a different way of browsing the web: some will use the navlinks, while others will look for the big shiny get started button). I don't think the extra buttons significantly affect either the design or the user interaction so I don't think a change is necessary here. What do you think?
> 
> Yup that makes sense! Instead would it be possible for the extra 'about' button to look like the 'Get Started' button and sit just the right of it, instead of with extra text under the image?

So this was intentional as well - it's about affordances really, so what the buttons look like they do should be in line with your expectations. Drawing from patterns I've personally picked up across comp sci websites is the 'get started' button is big, centered and boldly colored whereas the 'about' buttons are normally muted, smaller and off-center. I will try to find an example, but I feel the design direction here is consistent with similar websites

**Comment by CMorenoStokoe** [comment]
> > > The first part of the landing page had a lot of buttons, but some of them are the same (e.g. Get Started is also the User Guide, and there are two 'About' buttons, as well as the click down arrow which also takes you to the 'About' section). Could the navigation bar the top right hand corner be displayed in the center of that tile of text instead to avoid repeating buttons? Not sure what you think!
> > 
> > 
> > Hi Nina, thanks. Redundancy is an important part of design because you account for different user modalities (everyone has a different way of browsing the web: some will use the navlinks, while others will look for the big shiny get started button). I don't think the extra buttons significantly affect either the design or the user interaction so I don't think a change is necessary here. What do you think?
> 
> Yup that makes sense! Instead would it be possible for the extra 'about' button to look like the 'Get Started' button and sit just the right of it, instead of with extra text under the image?

How about this?
image

**Comment by ninadicara** [comment]
> > > > The first part of the landing page had a lot of buttons, but some of them are the same (e.g. Get Started is also the User Guide, and there are two 'About' buttons, as well as the click down arrow which also takes you to the 'About' section). Could the navigation bar the top right hand corner be displayed in the center of that tile of text instead to avoid repeating buttons? Not sure what you think!
> > > 
> > > 
> > > Hi Nina, thanks. Redundancy is an important part of design because you account for different user modalities (everyone has a different way of browsing the web: some will use the navlinks, while others will look for the big shiny get started button). I don't think the extra buttons significantly affect either the design or the user interaction so I don't think a change is necessary here. What do you think?
> > 
> > 
> > Yup that makes sense! Instead would it be possible for the extra 'about' button to look like the 'Get Started' button and sit just the right of it, instead of with extra text under the image?
> 
> How about this?
> image

That's perfect! I think it was more that having the text under the image was slightly throwing off the balance of that section visually - having the button slightly lighter coloured looks really good :)

**Pull request merged**